benefits/untracked_holiday.md

```diff
@@ -4,22 +4,32 @@ We're building a high-performance organisation that delivers great software prod
 
 We have adopted an untracked holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. 
 
-All we ask is that you follow a few guidelines:
+## Who approves holiday?
 
-* We use a shared Google Calendar to track who's away on holiday. Once booked, please check that your holiday is recorded in here. This calendar is used when working out project allocation and the number of days being billed to customers, so it's important that it is always up-to-date and accurate. [View our holiday calendar](https://www.google.com/calendar/ical/madebymade.co.uk_83gtf1a3s2ecshli2o6ub4kpmk%40group.calendar.google.com/public/basic.ics)
+We want to ensure teams own their holidays and that includes thinking about the commercial impact of taking them. To that affect we need to make sure you've done the following before booking holiday:
 
-* To book holiday, just send an email to the most appropriate person and include the dates of that you require.
+* Ask your team mates if it's okay with them
+* Ask your customers if it's okay with them
+* Ensure your responsibilities are covered whilst you're away
+* Ensure that the holiday is placed in the [holiday calendar](https://www.google.com/calendar/ical/madebymade.co.uk_83gtf1a3s2ecshli2o6ub4kpmk%40group.calendar.google.com/public/basic.ics)
 
-* If you're only taking a few days off, this can be approved by one of your sprint or support team mates.
+## How long are you taking?
 
-* If you're taking more than one week off, this should be approved by Chris or Rory at least one month in advance. 
+We trust you'll use your best judgement when taking holidays but in case you want some guidance on the kind of notice we expect when you take your holiday:
 
-* Having several team members away at the same time can make life really difficult for everyone else. Try to forward-plan your holiday, so it doesn't have a detrimental affect on your team mates or on the company.
+* **Less than 1 week:**<br>We recommend giving at least 2 weeks notice for a 1 day off and closer to 4 weeks notice for more
+* **Between 1 and 2 weeks:**<br>We recommend giving at least 4 weeks notice for this kind of holiday, ideally more.
+* **Greater than 3 weeks:**<br>Please try and provide at least three months notice, so we can prepare for your absence.
+
+## Additional considerations
 
-* If you're taking three (or more) weeks off, please try and provide at least three months notice, so we can prepare for your absence. 
+* We use the calendar when working out project allocation and the number of days being billed to customers, so it's important that it is always up-to-date and accurate.
 
 * Employees will not “accrue” holiday days under this policy, therefore you will not receive compensation for “unused” holiday time when you leave.  
```

Comment:
Contractual obligations don't exactly mirror the policy in practice, so this isn't strictly true